rhv4/profiles/rhvh-stig.profile

```diff
@@ -0,0 +1,456 @@
+    - install_hids
+    - ldap_client_start_tls
+    - package_aide_installed
+    - package_dracut-fips_installed
+    - rpm_verify_hashes
+    - install_PAE_kernel_on_x86-32
+    - service_kdump_disabled
+    - sysctl_fs_suid_dumpable
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_exec_shield
+    - sysctl_kernel_randomize_va_space
+    - var_account_disable_post_pw_expiration=35
+    - var_accounts_max_concurrent_login_sessions=10
+    - var_accounts_maximum_age_login_defs=60
+    - var_accounts_minimum_age_login_defs=7
+    - var_accounts_password_minlen_login_defs=6
+    - var_accounts_password_warn_age_login_defs=7
+    - var_accounts_tmout=10_min
+    - var_password_pam_difok=8
+    - var_password_pam_maxclassrepeat=4
+    - var_password_pam_minclass=4
+    - var_password_pam_unix_remember=5
+    - account_disable_post_pw_expiration
+    - accounts_max_concurrent_login_sessions
+    - accounts_maximum_age_login_defs
+    - accounts_minimum_age_login_defs
+    - accounts_password_pam_difok
+    - accounts_password_pam_maxclassrepeat
+    - accounts_password_pam_minclass
+    - accounts_password_pam_unix_remember
+    - accounts_password_warn_age_login_defs
+    - accounts_tmout
+    - banner_etc_issue
+    - display_login_attempts
+    - set_password_hashing_algorithm_libuserconf
+    - set_password_hashing_algorithm_logindefs
+    - set_password_hashing_algorithm_systemauth
+    - package_opensc_installed
+    - var_smartcard_drivers=cac
+    - configure_opensc_nss_db
+    - configure_opensc_card_drivers
+    - force_opensc_card_drivers
+    - service_pcscd_enabled
+    - sssd_enable_smartcards
+    - sssd_memcache_timeout
+    - sssd_offline_cred_expiration
+    - sssd_ssh_known_hosts_timeout
+    - encrypt_partitions
+    - ensure_redhat_gpgkey_installed
+    - ensure_gpgcheck_globally_activated
+    - ensure_gpgcheck_never_disabled
+    - ensure_gpgcheck_repo_metadata
+    - ensure_gpgcheck_local_packages
+    - network_sniffer_disabled
+    - network_ipv6_disable_rpc
+    - network_ipv6_privacy_extensions
+    - sshd_enable_x11_forwarding
+    - clean_components_post_updating
+    - kernel_module_cramfs_disabled
+    - kernel_module_dccp_disabled
+    - kernel_module_freevxfs_disabled
+    - kernel_module_hfs_disabled
+    - kernel_module_hfsplus_disabled
+    - kernel_module_jffs2_disabled
+    - kernel_module_sctp_disabled
+    - var_sssd_ssh_known_hosts_timeout=5_minutes
+    - var_accounts_minimum_age_login_defs=1
+    - var_password_pam_maxrepeat=3
+    - var_account_disable_post_pw_expiration=0
+    - var_removable_partition=dev_cdrom
+    - var_time_service_set_maxpoll=system_default
+    - sysctl_net_ipv4_conf_all_accept_source_route_value=disabled
+    - sysctl_net_ipv6_conf_all_accept_source_route_value=disabled
+    - rpm_verify_ownership
+    - accounts_password_pam_maxrepeat
+    - accounts_password_set_min_life_existing
+    - accounts_password_set_max_life_existing
+    - sudo_remove_nopasswd
```

Comment:
@shawndwells should we remove this rule or add a new rule that has an exception for `ovirt-deploy`

Reply:
likely create a rule w/an exception

Reply:
@shawndwells done... new rule is `sudo_vdsm_nopasswd`